cdap-ui/app/cdap/components/DataPrep/Directives/Calculate/index.js

```diff
@@ -42,7 +42,7 @@ export default class Calculate extends Component {
   constructor(props) {
     super(props);
 
-    this.NUMBER_TYPES = ['integer', 'short', 'long', 'float', 'double'];
+    this.NUMBER_TYPES = NUMBER_TYPES;
```

Comment:
if it's just the constant, is it need to be assigned to `this`?

Reply:
Good point. Will fix this.